tests/ui/find_is_some_on_strs.stderr

```diff
@@ -0,0 +1,58 @@
+error: called `is_some()` after calling `find()` on a string. This is more succinctly expressed  by calling `contains()`.
+  --> $DIR/find_is_some_on_strs.rs:15:27
+   |
+LL |     let _ = "hello world".find("world").is_some();
+   |                           ^^^^^^^^^^^^^^^^^^^^^^^ help: try this: `contains("world")`
```

Comment:
I think having two `:` looks weird, how about?
```
help: try `contains("world")`
```

Reply:
I agree, although I don't know if I have control over it.  The extra `:` is added by `span_lint_and_sugg()`.  I call it with `"try this"` for the corresponding argument (without `:`).